src/services/conversionRates/getConversionRatesService.js

```diff
@@ -198,22 +198,21 @@ const getHourlyUSDRateCoingecko = async (rateSymbol, timestampMS, coingeckoId =
 
   return rate;
 };
-
-const getHourlyUSDRateCryptocompare = async (timestamp, tokenSymbol) => {
+const getHourlyRateCryptocompare = async (timestamp, fromToken, toToken) => {
   const timestampMS = Math.round(timestamp / 1000);
 
   const resp = JSON.parse(
     await rp(
-      `https://min-api.cryptocompare.com/data/histohour?fsym=${tokenSymbol}&tsym=USD&toTs=${timestampMS}&limit=1`,
+      `https://min-api.cryptocompare.com/data/histohour?fsym=${fromToken.symbol}&tsym=${toToken.symbol}&toTs=${timestampMS}&limit=1`,
     ),
   );
 
   const tsData = resp && resp.Data && resp.Data.find(d => d.time === timestampMS);
 
   if (!tsData) throw new Error(`Failed to retrieve cryptocompare rate for ts: ${timestampMS}`);
-
-  return ((tsData.high + tsData.low) / 2).toFixed(2);
-};
+  let decimals = toToken && toToken.decimals ? toToken.decimals : 2;
```

Comment:
According to convention, `const` variable type is better candidate. Please use `let` when the variable will be modified after.